README.md

```diff
@@ -1 +1,13 @@
-# governance
+# Governance
+
+Founders & Coders C.I.C. is a social enterprise whose main purpose is to develop communities capable of supporting innovative and accessible forms of learning. 
```

Comment:
Could we say something like 'Our most immediate aim is to make learning to code accessible to everyone' (in better terms) so that there's something more tangible and less lofty that people in F&C at the moment can identify with? Or do you not think this is necessary?